inc/class_parser.php

```diff
@@ -635,11 +635,19 @@ function parse_badwords($message, $options=array())
 					$badword['replacement'] = "*****";
 				}
 
-				// Take into account the position offset for our last replacement.
-				$badword['badword'] = str_replace('\*', '([a-zA-Z0-9_]{1})', preg_quote($badword['badword'], "#"));
+				if($badword['regex'])
+				{
+					//_dump($badword['badword']);
```

Comment:
Can you remove this comment please? Looks like it was left behind from debugging 😄